src/utils/uint256.ts

```diff
@@ -18,6 +26,14 @@ export function uint256ToBN(uint256: Uint256) {
 /**
  * Test BigNumberish is smaller or equal 2**256-1
```

Comment:
This claim is not accurate, the same for the claim below
Claim is (-infinite <= 2^256-1)
**accurate claim is:**
Test BigNumberish is in the range [0, 2^256-1]
https://stackoverflow.com/questions/4396290/what-does-this-square-bracket-and-parenthesis-bracket-notation-mean-first1-last

Reply:
Thank you for pointing this out. This description came with the old version.
I have implemented the correct thing now.